src/main.js

```diff
@@ -1,6 +1,10 @@
 const path = require('path');
 const { app, BrowserWindow, shell, ipcMain, dialog } = require('electron');
 const { downloadMemories } = require('./memoryDownloader');
+const dayjs = require('dayjs');
+const utc = require('dayjs/plugin/utc');
+
+dayjs.extend(utc);
```

Comment:
Does calling `dayjs.extend(utc);` alter the `dayjs` import in other files?

Even so, I only see `dayjs` imported and used in `src/services/exif.js`. Is there a reason to not move this into that file?

Reply:
1. yes, it adds support for utc across all dayjs instances in the project using prototypal inheritance. so any instance imported will have utc working.
2. yes, as the extend needs to be done once and it'll work for every file and secondly, I think since, I'll be picking up the dayjs conversion issue which will also use this code in the future.